lib/livebook/teams/broadcasts.ex

```diff
@@ -67,6 +68,14 @@ defmodule Livebook.Teams.Broadcasts do
     broadcast(@clients_topic, {:client_connected, id})
   end
 
+  @doc """
+  Broadcasts under `#{@clients_topic}` topic when hub was deleted and the client disconnected.
```

Comment:
Should this be hub deleted and put it Livebook.Broadcasts? Because client disconnected makes me think that it may reconnect again.

Reply:
Or may be already emit a hub deleted event and we should use that instead?

Reply:
I've added this topic only for testing, but we need to know when the hub was deleted to remove the hub from all sessions

Reply:
We use it here as well, no? https://github.com/livebook-dev/livebook/pull/2573/files#diff-a6a42783f58f278f4e7f8482bd9f8c662143c977eb1d49dec96328e22b42b90cR1858 I am thinking if this event should be "hub deleted" instead?

Reply:
We already broadcast the hub has been deleted (`hubs_changed`), but the Team Client didn't disconnected yet, because we wait the GenServer to finish all his work before terminating it.

So, it would be related only to the GenServer (`TeamClient`), that was the reason to use `client_connected` (when the TeamClient finished the pipeline of all data received from `User/AgentConnected` events) and `client_disconnected` (when we already sent to the Supervisor to terminate the child).

WDYT?

Reply:
> We use it here as well, no?

Yeah, but thinking more about, Perhaps `hubs_changed` should be enough?

Reply:
Yes, exactly. We should disconnect but here we should only worry about the hubs deletion. Even if it takes 30 seconds that's fine. :)